src/main/java/requests/RequestManager.java

```diff
@@ -23,7 +23,7 @@
      * Declaration of different constructors to fit different request structures.
      */
     public RequestManager(String organizationName) {
-        this.organizationName = organizationName;
+        this.organizationName = organizationName.replaceAll("\\s+","").toLowerCase();
```

Comment:
Thanks! Please notice the other constructors.
Export it to a separate method and call it on each constructor.